boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh

```diff
@@ -28,17 +28,13 @@ mkdir $HARNESS_DIR
 
 echo "
 # THIS FILE IS GENERATED BY BOILERPLATE. DO NOT EDIT.
-FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
-
-ENV PKG=/go/src/github.com/openshift/${OPERATOR_NAME}/
-WORKDIR \${PKG}\
-
+FROM registry.ci.openshift.org/openshift/release:golang-1.19 AS builder
+WORKDIR /go/src/github.com/openshift/aws-vpce-operator/
```

Comment:
`aws-vpce-operator` ?

Reply:
This is appearing because `ocm-agent-operator` is partially onboarded into boilerplate-osde2e, so it's pulling in the boilerplate osde2e related components - where this appears to be statically hard-coded [0]

That's an issue for boilerplate to eventually fix when it eventually finishes onboarding OAO, but in terms of this boilerplate update it is "correct" that it appears, and should not cause issues.

[0] https://github.com/openshift/boilerplate/blob/b8a9cf4696030c34ff43e7dd8fe5e88c6326695a/boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh#L32

Reply:
Ack thanks for the clarification Matt!